source/features/previous-next-commit-buttons.tsx

```diff
@@ -9,10 +10,14 @@ function init(): false | void {
 		return false;
 	}
 
-	const previousNext = originalPreviousNext.cloneNode(true);
-	const files = select('#files')!;
+	// Wrap the button in a <div> to avoid #4503
+	select('#files')!.after(
+		<div className="d-flex flex-justify-end">
+			{originalPreviousNext.cloneNode(true)}
+		</div>,
+	);
 
-	files.after(previousNext);
+	select('#files ~ .flash-warn')?.classList.add('mt-3');
```

Comment:
Not sure if the class should be applied to the wrapping `div` instead.

Reply:
If it works there yes. It shortens this file by 2 lines right?